packages/fixed-bridge/src/BridgeSide.ts

```diff
@@ -54,8 +54,8 @@ export class BridgeSide {
   *** make its way to the neighbor root of the linked anchor on chain X.
   *** @param linkedAnchorInstance: the anchor instance on the opposite chain
   ***/
-  public async createAnchorUpdateProposalData(linkedAnchorInstance: Anchor) {
-    const proposalData = await linkedAnchorInstance.getProposalData();
+  public async createAnchorUpdateProposalData(linkedAnchorInstance: Anchor, thisAnchorInstance: Anchor) {
```

Comment:
This isn't a good naming scheme, lets avoid `thisAnchorInstance` please. Propose other names if you're unsure.